dungeon/src/contrib/systems/HealthBarSystem.java

```diff
@@ -123,5 +125,6 @@ private void updatePosition(ProgressBar pb, PositionComponent pc) {
     pb.setPosition(screenPosition.x, screenPosition.y);
   }
 
-  private record EnemyData(HealthComponent hc, PositionComponent pc, ProgressBar pb) {}
+  private record EnemyData(
+      HealthComponent hc, PositionComponent pc, DrawComponent dc, ProgressBar pb) {}
```

Comment:
@Flamtky diese vier components sollten dann auch alle (in dieser reihenfolge) als filter im ctor gesetzt und auch im filterstream genutzt werden. kannst du das bitte noch ergänzen?

Reply:
ok, **drei** components :)

Reply:
👍. Habe es auch schön alphabetisch sortiert 😅